### PR TITLE
Feature/checkpoints improvement

### PR DESCRIPTION
**Pull request by jeremyletang**

### CODE REVIEW

File: non-protocol-specs/0005-NP-LIMN-limited_network_life.md [hunk 1]
```diff
@@ -33,12 +33,23 @@ Information to store:
...
 - Event ID of the last processed deposit event for all bridged chains
 - Withdrawal transaction bundles for all bridged chains for all ongoing withdrawals (parties with non-zero "signed-for-withdrawal" balances)
 - hash of the previous block, block number and transaction id of the block from which the snapshot is derived
+- ERC20 collateral:
+  - last block height of a confirmed erc20 deposit on the ethereum chain with `number_of_confirmations`.
+  - all pending erc20 deposits (not confirmed before this block)
+- Staking:
+  - last block of a confirmed stake_deposit on the staking contract on the ethereum chain with `number_of_confirmations`.
+  - last block of a confirmed stake_deposit on the vesting contract on the ethereum chain with `number_of_confirmations`.
+  - all the staking events from both contracts
```
Comment (davidsiska-vega) [inline non-protocol-specs/0005-NP-LIMN-limited_network_life.md]: Can we link to the relevant specs here please?

**Review by davidsiska-vega — COMMENTED** [review]
Apart from that one nit-pick comment it looks fine to me.

**Comment by jeremyletang** [comment]
@davidsiska-vega just addressed your comments here.

**Review by witgaw — COMMENTED** [review]
LGTM (Gordon wanted me to take a look).

**Review by witgaw — DISMISSED** [review]
LGTM (now with approval)

**Review by witgaw — APPROVED** [review]
LGTM

**Pull request merged**